### PR TITLE
Videos listing and filter

### PR DESCRIPTION
**Pull request by Sharmaryan**
- created video listing page
- user can sort videos based on categories
- user can select category from homepage

### CODE REVIEW

File: src/components/Categories/Categories.js [hunk 1]
```diff
@@ -1,57 +1,61 @@
 import React from "react";
 import "./Categories.css";
 import { Link } from "react-router-dom";
-import { useTheme } from "../../context/theme-context/theme-context";
+
+import { useVideo } from "../../context/video-context/video-context";
```
Comment (arpit01923) [inline src/components/Categories/Categories.js]: try absolute path to avoid ../../

File: src/components/Categories/Categories.js [hunk 2]
```diff
@@ -1,57 +1,61 @@
...
 
 export const Categories = () => {
   const categories = [
```
Comment (arpit01923) [inline src/components/Categories/Categories.js]: this category array should be put inside database and fetch it here

File: src/components/Sidebar/Sidebar.jsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import "./Sidebar.css";
+import { Link } from "react-router-dom";
+import { BiLike, BiPlayCircle, BiHome, BiHistory } from "react-icons/bi";
```
Comment (arpit01923) [inline src/components/Sidebar/Sidebar.jsx:4]: is there react icons also

**Review by pradipgharge — APPROVED** [review]
code is clean and looks good to merge

File: src/components/VideoCards/VideoCards.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import { VideoCard } from "../VideoCard/VideoCard";
+import { useVideo } from "../../context/video-context/video-context";
+import { selectCategory } from "../../utils/selectCategory";
```
Comment (arpit01923) [inline src/components/VideoCards/VideoCards.jsx]: try absolute paths

**Review by arpit01923 — APPROVED** [review]
- clean UI
- good code implementation
- left suggestion, maybe useful

**Pull request merged**